src/themes/wix/variables.st.css

```diff
@@ -32,11 +32,16 @@
 	P20: #cd68ed;
     GR20: #dadada;
 
-    /* Inputs */
-    fontStyle: 16px "HelveticaNeue-Regular", sans-serif;
+    /* Typography */
 
+    h1FontStyle: 36px "HelveticaNeueW01-Thin", sans-serif;
```

Comment:
are you sure about this? it isn't used anywhere as far as I see except selector `h1`

Reply:
Took it from the guide
https://app.zeplin.io/project/5864e02695b5754a69f56150/screen/58650608493286fa25d9382a